src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -357,8 +357,8 @@ const Notebooks = _.flow(
             style: {
               ...Style.elements.card.container, height: 125, marginTop: 15
             },
-            disabled: appIsSettingUp(app) || appIsDeleting(app),
-            tooltip: (appIsSettingUp(app) || appIsDeleting(app)) ? `Your Galaxy app is being ${appIsSettingUp(app) ? 'created' : 'deleted'}` : undefined,
+            disabled: appIsSettingUp(app),
+            tooltip: appIsSettingUp(app) ? 'Your Galaxy app is being created' : undefined,
```

Comment:
Opt Nit: Any reason to use a ternary here over an AND? (`appIsSettingUp(app) && 'Your Galaxy app is being created'`)